akka-stream/src/main/scala/akka/stream/impl/fusing/AggregateWithTimeBoundary.scala

```diff
@@ -72,28 +64,24 @@ class AggregateWithBoundary[T, Agg, Emit](
         val input = grab(in)
         if (aggregated == null) aggregated = allocate
         if (aggregate(aggregated, input)) harvestAndEmitOrEnqueue()
-        if (!bufferFull) pull(in)
+        if (!backPressure) pull(in)
```

Comment:
Good, this should let it continue consuming elements from upstream until the aggregate is "full" and only then propagate downstream backpressure.

Reply:
Somehow this is referring to older code. now it'a `if (!isAvailable(out)) pull(in)` to be more explicit, since out is the only signal for downstream backpressure.
Note this requires a onPush in the first place, and will stop the pull once aggregate is emitted.